### PR TITLE
Add release announcement for 3.10

### PR DESCRIPTION
**Pull request by melcorr**
will proofread and check links again in the morning

### CODE REVIEW

File: _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+fields = {
+         "id": ["exact", "in"],
+         "name": ["exact", "iexact", "in", "contains", "icontains"],
+     }
+```
+
+
+## Add your own labels to objects
+
+With the addition of the `pulp_labels` field, you can create labels to add your own key/data for your resources in Pulp. You can also modify and remove these labels if needed.
```
Comment (daviddavis) [inline _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md:76]: This section looks good. Can you link to the new doc I created?

https://docs.pulpproject.org/pulpcore/workflows/labels.html

Comment (melcorr) [reply _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md:76]: Gladly

File: _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+`MEDIA_ROOT` is now its own directory. This change has the advantage of limiting the file permissions and the SELinux file contexts in a shared Pulp 2 and Pulp 3 deployment. This also adds compatibility with the `django_extensions' unreferenced_files` command, which lists all files in `MEDIA_ROOT` that are not in the database.
+
+Other paths are kept on the same absolute paths.
+
+The settings and logging documentation has been updated to reflect these changes. [#7178](https://pulp.plan.io/issues/7178)
+
+## New generic list endpoint
+
+This release includes a new endpoint that you can use to list all `Content`, `ContentGuards`, and `Repositories`. [#7204](https://pulp.plan.io/issues/7204)
```
Comment (mdellweg) [inline _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]:
```suggestion
This release includes new endpoints that you can use to list all `Content`, `ContentGuards`, and `Repositories`. [#7204](https://pulp.plan.io/issues/7204)
```
It's three of them to this day. ;)

File: _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+## Update to the SigningService
+
+A new `public_key` field has been added to the **SigningService**. This field preserves the value of the public key. A new `pubkey_fingerprint` field was introduced as well, which identifies the
+public key. [#7700](https://pulp.plan.io/issues/7700)
+
+As a result of this, the following deprecation notice has been added for this release:
+
+`The path to the public key is no longer required to be present in the output of a signing script. It is recommended to pass the value of the public key during the object creation process instead.`
```
Comment (mdellweg) [inline _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]:
```suggestion
`The path to the public key is no longer required to be present in the output of a signing script. It is required to pass the value of the public key during the object creation process instead.`
```

I think it's not an option. @bmbouter can you verify?

Comment (melcorr) [reply _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]: For this blog @mdellweg I just took what message was in the code. The change would have to happen elsewhere?

Comment (mdellweg) [reply _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]: I'm seeing this one:
https://github.com/pulp/pulpcore/commit/2e0366013d2035c96156bf83debdfc7c03f647a0#diff-81f6a78175bb93934b6beff952646d3ca1ef3731f1ff14492d4ec77bfc3fdf82R652

Comment (melcorr) [reply _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]: Will amend that now!

File: _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+This will be useful for:
+
+* Users to see how a system's RBAC has been modified
+* Plugin writers who want to ship changes or bugfixes to default policies they previously shipped and don't want to overwrite a user's custom policy.
+
+## Migration imports now use the `apps.get_model`
+In various places in the migrations, models are imported directly and then used. This was a hazard because when the model definition changes, this code can break.
+
+The data migrations now use `apps.get_model(...)` instead. [#8203](https://pulp.plan.io/issues/8203)
```
Comment (goosemania) [inline _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]: Not sure if it's worth calling this out. It's not a user facing item and not plugin API fix/feature as well.

Comment (melcorr) [reply _posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md]: I will remove then!

**Review by goosemania — APPROVED** [review]
Thanks!

**Pull request merged**